test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -5980,14 +5981,30 @@ def submit(self, obj, script=None, extend=None, submit_dir=None):
             os.chdir(submit_dir)
         c = None
         # 1- Submission using the command line tools
+        runcmd = []
+        exp_str = ""
+        if env:
+            runcmd += ['#!/bin/bash\n']
+            for k, v in env.items():
+                if '()' in k:
+                    f_name = k.replace('()', '')
+                    runcmd += [k, v, "\n", "export", "-f", f_name]
+                else:
+                    if 'export' not in runcmd:
```

Comment:
can you explain here why you are checking 'export' in runcmd instead k?

Reply:
I am passing {"Key":"value", "k2":"v2",...}, export command is added to runcmd. e.g: ['export', 'k1=v1']. So, i am checking here if export command is already present in runcmd.
Thanks for pointing this out i will add a comment here.